vitest.config.ts

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    coverage: {
+      branches: 80,
+      functions: 80,
+      lines: 90,
+      statements: 90,
+      reporter: 'text',
+    },
+    exclude: ['**/dist/**', '**/node_modules/**', '**/__fixtures__/**', '**/helpers/**'],
+    globals: true,
+    globalSetup: ['./__tests__/setup.js'],
+    setupFiles: ['./__tests__/helpers/vitest.matchers.ts'],
+
+    /**
+     * We can't run tests with `threads` on because we use `process.chrdir()` in some tests and
+     * that isn't available in worker threads, and it's way too much work to mock out an entire
+     * filesystem and `fs` calls for the tests that use it.
+     *
+     * @see {@link https://github.com/vitest-dev/vitest/issues/566}
+     */
+    threads: false,
```

Comment:
It might be possible to refactor the `process.chdir()` stuff away and instead swap in the working directory we're changing into into the paths we're trying to look in? Something like this with `readdirRecursive`:

![Screen Shot 2023-08-10 at 12 46 44 AM](https://github.com/readmeio/rdme/assets/33762/87ddd182-3b7f-4048-9da5-aacd3e42cf2d)

If we get rid of `process.chdir()` we should easily be able to shave a minute+ off of the Vitest execution time.

Reply:
ah it's used as the `workingDirectory` parameter for resolving refs in our `openapi` family of commands — will look into it!

Reply:
sadly we can't move away from the `process.chdir()` stuff yet, we'll need to get rid the `workingDirectory` flag in the `openapi` family of commands. ticketed here: RM-7754